### PR TITLE
Setting a card_token on a customer, should update the card when the c…

### PR DESCRIPTION
**Pull request by excid3**
…ustomer is retrieved

### CODE REVIEW

**Comment by excid3** [comment]
cc @mickeytgl

File: lib/pay/billable/stripe.rb [hunk 1]
```diff
@@ -20,7 +20,9 @@ def update_stripe_card(token)
         token = ::Stripe::Token.retrieve(token)
 
         return if token.card.id == customer.default_source
-        save_stripe_card(token, customer)
+        result = save_stripe_card(token, customer)
+        self.card_token = nil
```
Comment (excid3) [inline lib/pay/billable/stripe.rb]: We want to reset the `card_token` attribute here after the token was updated on Stripe.

Comment (mickeytgl) [reply lib/pay/billable/stripe.rb]: Did this work out for you?? I tried to run this but again got an error. 
image

I tried to dive in to the code, but I'm getting pretty confused in the `save_stripe_card(token, customer)` and `update_stripe_card(token)` methods

Comment (excid3) [reply lib/pay/billable/stripe.rb]: I'm not sure what you mean?

**Pull request merged**